pkg/operation/botanist/secrets.go

```diff
@@ -127,8 +127,8 @@ func caCertConfigurations() []secretutils.ConfigInterface {
 		// generated to ensure that each CA has a unique common name. For backwards-compatibility, we still keep the
 		// CommonNames here (if we removed them then new CAs would be generated with the next shoot reconciliation
 		// without the end-user to explicitly trigger it).
-		&secretutils.CertificateSecretConfig{Name: v1beta1constants.SecretNameCACluster, CommonName: "kubernetes", CertType: secretutils.CACert},
 		&secretutils.CertificateSecretConfig{Name: v1beta1constants.SecretNameCAClient, CommonName: "kubernetes-client", CertType: secretutils.CACert},
+		&secretutils.CertificateSecretConfig{Name: v1beta1constants.SecretNameCACluster, CommonName: "kubernetes", CertType: secretutils.CACert},
```

Comment:
Is it supposed to be CAClinet?

Reply:
Why? The difference in this PR is that the config for `ca-client` is the first one (before `ca`) so that this code https://github.com/gardener/gardener/blob/e8999347e1367cdc9d108fd6b03446bd0a9040ad/pkg/utils/secrets/manager/generate.go#L149-L159
does not trigger.